docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+- | MR physics leader `César Caballero-Gaudes`_
+  |   Someone who can make sure calculations fit within our understanding of MR physics
+  |   Someone who can either answer MRI physics questions related to multi-echo or direct people to where they can find answers
+- | Processing algorithms leader `Eneko Uruñuela`_ (Decomposiiton) & `Dan Handwerker`_ (Metrics & Decision Tree)
+  |   Someone who can make sure algorithms are appropriately implemented (or knows enough to delegate to someone who can make sure implementation is good)
+  |   Someone who can either answer processing algorithm questions or direct people to where they can find answers
+- | Collaborative programming leader `Elizabeth DuPre`_ & `Joshua Teves`_
+  |   Helps make sure tedana is following best practices for Python code design, testing, and communication for issues/pull requests etc.
+- | Communications leader `Joshua Teves`_
+  |   Mailing list manager & other outward-facing communication about the project
+- | New contributors leader `Taylor Salo`_
+  |   Leads efforts to make contributor documentation more welcoming
+  |   Is a point of contact for potential contributors to make them feel welcome and direct them to relevant resources or issues
+- | Multi-echo fMRI support leader `Logan Dowdle`_
+  |   Monitors places where people may ask questions about tedana or multi-echo fMRI and tries to find someone to answer those questions
+- | Enforcer(s) of the `code of conduct`_ `Elizabeth DuPre`_ & `Dan Handwerker`_ & another volunteer
+  |   A person or people someone can go to if they want to report a code of conduct violation
+  |   If this is one person, that person should NOT be on the steering committee
+  |   If this is more than one person, at least one should not be on the steering committee
+  |   Ideal is someone who cares about tedana but DOESN’T know contributors well enough to say, ”Person X would never do that”
+
+
+
+Changing leaders
+````````````````
+Steering committee members can remove themselves from the steering committee at
+any time and open up a call for a new self-nomination. Anyone can request to take
+on a leadership role at any time. Once per year, there should be an explicit call
+to the larger contributor community asking if anyone wants to self nominate for
+membership on the steering committee or other leadership roles. If individuals
+cannot reach consensus on who steps back and who assumes new roles, then a
+majority vote of contributors from the previous 3 years will assign people to
+roles where there are conflicts.
+
+If there are concerns with a tedana steering committee member or leader, any
+enforcer of the code of conduct can ask anyone to step down from a leadership role.
+If a person refuses to step down, then an enforcer of the code of conduct can call
+a vote of contributors to remove an individual from a leadership role in tedana.
```

Comment:
Should it be a vote of all contributors ? I worry that this could mean putting out really personal information, depending on the code of conduct violation. I'd rather consensus among the CoC enforcers.

Reply:
I commented on this in the original doc as well, but I don't know the sweet spot. In an ideal world, a code of conduct enforcer will confront someone & ask them to step aside & they'd rather step aside than cause additional strife in the project. Forcing someone out only becomes an issue if the person refuses. If the forcing out process is limited to any subset of leaders then there's the risk of people sticking with their friend rather than the accuser. If we open it up completely, then it can be a privacy violation. Maybe we can give the accuser the power to escalate from CoC enforcers to Steering Committee to Contributors if they aren't believed in the smaller groups?

Reply:
I can't think of another organization where CoC disputes are brought to a public vote. I think it could turn out really badly for everyone, especially the victim. Honestly, I worry that folks would be less likely to report if that was even a possible outcome.

I understand your concern about viable checks, but I'd suggest this decision / discussion is left to the CoC enforcement team (and future PRs). Here, I'd just point to the enforcement team.

Reply:
I might even leave out the bit about a refusal to step down. That can be detailed in an Enforcement section of the CoC as it's agreed upon. I do think we could clearly list here, though, who the current Enforcement team includes.

Reply:
💯 agree with your concerns. I do want a clear Code of Conduct enforcement path if a very core project leader or a Code of Conduct enforcer is the violator. Those are the situations that tend to become the most ugly. I **really** hope this doesn't become an issue given the proposed CoC enforcers are me & @emdupre, but this is something that's worth future-proofing. I wonder there are existing models for CoC enforcement that account for these scenarios in for small projects like tedana. @KirstieJane?